src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+            penCircle.EndCap = System.Drawing.Drawing2D.LineCap.Round;
+
+            using System.Drawing.Font fontGauge = new(Font.Name, lineWidth * GaugeLabelsFontPct / 100, FontStyle.Bold);
+
+            lock (this)
+            {
+                int index;
+                for (int i = 0; i < numGroups; i++)
+                {
+                    // Data is reversed in case SingleGauge is selected
+                    // If OutsideToInside is selected, radius is reversed
+                    if (GaugeMode != RadialGaugeMode.SingleGauge)
+                    {
+                        index = i;
+                        gaugeRadius = (GaugeOrder == RadialGaugeOrder.InsideToOutside ? i + 1 : (numGroups - i)) * radiusSpace;
+                    }
+                    else
+                    {
+                        index = numGroups - i - 1;
+                    }
+
+                    // Set color values
+                    pen.Color = GaugeColors[index];
+                    penCircle.Color = LightenBy(GaugeColors[index], DimPercentage);
+
+                    // Draw gauge background
+                    if (GaugeMode != RadialGaugeMode.SingleGauge)
+                        gfx.DrawArc(penCircle, (origin.X - gaugeRadius), (origin.Y - gaugeRadius), (gaugeRadius * 2), (gaugeRadius * 2), _StartingAngleBackGauges, maxBackAngle);
+
+                    // Draw gauge
+                    gfx.DrawArc(pen, (origin.X - gaugeRadius), (origin.Y - gaugeRadius), (gaugeRadius * 2), (gaugeRadius * 2), (float)DataAngular[index, 0], (float)DataAngular[index, 1]);
+
+                    // Draw gauge labels
+                    if (ShowGaugeValues)
+                    {
+                        DrawTextOnCircle(gfx,
+                            fontGauge,
+                            labelBrush,
+                            new RectangleF(dims.DataOffsetX, dims.DataOffsetY, dims.DataWidth, dims.DataHeight),
+                            gaugeRadius,
+                            (float)DataAngular[index, 0],
+                            (float)DataAngular[index, 1],
+                            origin.X,
+                            origin.Y,
+                            DataRaw[index].ToString("0.##"),    // Could be set as a user-choice paremeter?
+                            _GaugeLabelPos);
+                    }
+
+                }
+
+            }
+
+        }
+
+        #region Color routines
+
+        /// <summary>Creates color with corrected brightness.</summary>
+        /// <param name="color">Color to correct.</param>
+        /// <param name="correctionFactor">The brightness correction factor. Must be between -1 and 1. 
+        /// Negative values produce darker colors.</param>
+        /// <returns>Corrected <see cref="Color"/> structure.</returns>
+        /// <seealso href="http://gist.github.com/zihotki/09fc41d52981fb6f93a81ebf20b35cd5"/>
+        private Color ChangeColorBrightness(Color color, float correctionFactor)
```

Comment:
I was aware that the Color object as a method returning hue, saturarion, and [brightness](https://docs.microsoft.com/es-es/dotnet/api/system.drawing.color.getbrightness?view=net-5.0)
The only function I could find in the .NET API to convert between HSL and RGB can be found in 
 [Microsoft.VisualStudio.Modeling.Diagrams](https://docs.microsoft.com/es-es/dotnet/api/microsoft.visualstudio.modeling.diagrams.hslcolor?view=visualstudiosdk-2019) (Microsoft.VisualStudio.Modeling.Sdk.15.0.dll)

I'm not sure it would be a good idea to include that assembly reference into the project.